pandas/core/groupby/ops.py

```diff
@@ -47,6 +47,7 @@
     ensure_uint64,
     is_1d_only_ea_dtype,
 )
+from pandas.core.dtypes.inference import is_array_like
```

Comment:
can we use a specific isinstance check instead?

Reply:
Ah yes, we can go the other way round